app/Http/Controllers/JobController.php

```diff
@@ -313,6 +313,35 @@ function ($skillLevel) use ($skillLangs) {
      */
     public function store(Request $request, JobPoster $jobPoster = null)
     {
+        // Job Poster validation rules
+        $validatedData = $request->validate([
```

Comment:
This validation needs to run only when the poster is being submitted. You should be able to save an incomplete draft.

Reply:
Only runs on publish now. Could use your help moving the rules to JobPosterValidator or someone else more appropriate.